README.rst

```diff
@@ -180,10 +180,11 @@ execution using the familiar python ``in`` operator
 
 .. code-block:: python
 
-    if job_instance in scheduler:
+    if job_instance in list_of_job_instances:
```

Comment:
`if job_instance in scheduler` is correct, scheduler implements `__contains__` dunder method.